### DISCUSSION TITLE
Only use partial chain for effect

### DISCUSSION THREAD

**Discussion by Tomadoggy** [digitalninja-ro/klipper-neopixel#13] [Q&A] [answered]
How would I configure/alter a macro to only use (by example) LED's 1 through 15 of a 30 LED chain_count?  

I have a strip at the front of my printer and during startup I want to use 1/2 of this to display progress for extruder and the other half to display bed heat up.

**Comment by digitalninja-ro** [comment, marked as answer]
Hi. My macro doesn't support partial leds array because I want to keep it as simple as possible to use. If you want to use just parts of leds I recommend  https://github.com/BrendaIT/klipper-led_effect

**Comment by Tomadoggy** [comment]
Thank you for the answer!
